en/docs/faqs.md

```diff
@@ -117,22 +143,22 @@ Instead of using the `                 micro-gw                ` command, naviga
 
```

Comment:
this block comes under a code highlighted section in the preview. like below. can you correct that too?
```md
    ### Building a Microgateway project
```